prow/scripts/cluster-integration/kyma-integration-k3d.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+export KYMA_SOURCES_DIR="./kyma"
+
+function prereq_test() {
+  command -v node >/dev/null 2>&1 || { echo >&2 "node not found"; exit 1; }
+  command -v npm >/dev/null 2>&1 || { echo >&2 "npm not found"; exit 1; }
+  command -v jq >/dev/null 2>&1 || { echo >&2 "jq not found"; exit 1; }
+  command -v helm >/dev/null 2>&1 || { echo >&2 "helm not found"; exit 1; }
+  command -v kubectl >/dev/null 2>&1 || { echo >&2 "kubectl not found"; exit 1; }
+}
+
+function load_env() {
+  ENV_FILE=".env"
+  if [ -f "${ENV_FILE}" ]; then
+    export $(xargs < "${ENV_FILE}")
+  fi
+}
+
+function prepare_k3d(){
+    wget -q -O - https://raw.githubusercontent.com/rancher/k3d/main/install.sh | TAG=v4.4.7 bash
```

Comment:
Consider preparing the OS image with already built-in k3d binary, it will speed up a bit the execution time.

Reply:
To add it to our VM image you need to modify this script: https://github.com/kyma-project/test-infra/blob/main/development/custom-image/install-deps-debian.sh
It will be automatically used in every job after you merge change in this file to the main branch.

Reply:
Perfect, thank you!
Added the k3d dependency to the custom image here. Can you have a look?